docs/topics/compatibility-guide-15.md

```diff
@@ -0,0 +1,274 @@
+> **Short summary**: Kotlin 1.5 will change the order of arguments execution in calls with named vararg
+>
+> **Deprecation cycle**:
+>
+> - < 1.5: introduce warning for the problematic constructors
+> - \>= 1.5: raise this warning to an error,
+>  `-XXLanguage:-UseCorrectExecutionOrderForVarargArguments` can be used to temporarily revert to pre-1.5 behavior
+
+### Use default value of the parameter in operator functional calls
+
+> **Issue**: [KT-42064](https://youtrack.jetbrains.com/issue/KT-42064)
+>
+> **Component**: Kotlin/JVM
+>
+> **Incompatible change type**: behavioral
+>
+> **Short summary**: Kotlin 1.5 will use default value of the parameter in operator calls
+>
+> **Deprecation cycle**:
+>
+> - < 1.5: old behavior (see details in the issue)
+> - \>= 1.5: behavior changed,
+>  `-XXLanguage:-JvmIrEnabledByDefault` can be used to temporarily revert to pre-1.5 behavior
+
+### Produce empty reversed progressions in for loops if regular progression is also empty 
+
+> **Issue**: [KT-42533](https://youtrack.jetbrains.com/issue/KT-42533)
+>
+> **Component**: Kotlin/JVM
+>
+> **Incompatible change type**: behavioral
+>
+> **Short summary**: Kotlin 1.5 will produce empty reversed progressions in for loops if regular progression is also empty
+>
+> **Deprecation cycle**:
+>
+> - < 1.5: old behavior (see details in the issue)
+> - \>= 1.5: behavior changed,
+>  `-XXLanguage:-JvmIrEnabledByDefault` can be used to temporarily revert to pre-1.5 behavior
+
+### Straighten Char-to-code and Char-to-digit conversions out
+
+> **Issue**: [KT-23451](https://youtrack.jetbrains.com/issue/KT-23451)
+>
+> **Component**: kotlin-stdlib
+>
+> **Incompatible change type**: source
+>
+> **Short summary**: Since Kotlin 1.5, conversions of Char to number types will be deprecated
+>
+> **Deprecation cycle**:
+>
+> - 1.5: deprecate `Char.toInt()/toShort()/toLong()/toByte()/toDouble()/toFloat()` and the reverse functions like `Int.toChar()`, and propose replacement 
+
+### Inconsistent case-insensitive comparison of characters in kotlin.text functions
+
+> **Issue**: [KT-45496](https://youtrack.jetbrains.com/issue/KT-45496)
+>
+> **Component**: kotlin-stdlib
+>
+> **Incompatible change type**: behavioral
+>
+> **Short summary**: Since Kotlin 1.5, `Char.equals` will be improved in case-insensitive case by first comparing whether the uppercase variants of characters are equal, then whether the lowercase variants of those uppercase variants (as opposed to the characters themselves) are equal
+>
+> **Deprecation cycle**:
+>
+> - < 1.5: old behavior (see details in the issue)
+> - 1.5: change behavior for `Char.equals` function 
+
+### Remove default locale-dependent case conversion API
+
+> **Issue**: [KT-43023](https://youtrack.jetbrains.com/issue/KT-43023)
+>
+> **Component**: kotlin-stdlib
+>
+> **Incompatible change type**: source
+>
+> **Short summary**: Since Kotlin 1.5, default locale-dependent case conversion functions like `String.upperCase()` will be deprecated
```

Comment:
```suggestion
> **Short summary**: Since Kotlin 1.5, default locale-sensitive case conversion functions like `String.upperCase()` will be deprecated
```